### PR TITLE
Fix 1423 Prevent emoji picker clicks from bubbling up

### LINKED ISSUE #1423: [Web] Emoji picker closes instantly
**Describe the bug**
If I open the emoji picker and click in the search field, for example, it closes immediately

**To Reproduce**

1. open the emoji picker on web and click in the search field

**Expected behavior**

I expect to be able to enter a search term or access the emoji categories like I was able to do in 1.48 before the scroll bug was fixed

**Details**

 - Platform: Windows 11
 - Platform version: Firefox 117.0
 - App version: 1.49

### PR DESCRIPTION
**Pull request by ansh**
Similar to the composer or the modal, we want to prevent Emoji Picker clicks from bubbling up!

Fixes #1423

### CODE REVIEW

**Comment by gaearon** [comment]
Is there some logic in some ancestor that closes it on click?

If yes, instead of preventing propagation here, have we considered doing a `node.contains` check on `e.target` there, and not closing if the target is within the picker?

File: src/view/com/composer/text-input/web/EmojiPicker.web.tsx [hunk 1]
```diff
@@ -58,21 +58,27 @@ export function EmojiPicker({close}: {close: () => void}) {
     // eslint-disable-next-line react-native-a11y/has-valid-accessibility-descriptors
     <TouchableWithoutFeedback onPress={close} accessibilityViewIsModal>
```
Comment (ansh) [inline src/view/com/composer/text-input/web/EmojiPicker.web.tsx:59]: @gaearon The ancestor is simply just this one.

I was following the pattern in the `Composer.web.tsx` file and the `Modal.web.tsx` file, but your idea sounds good. 

Let me take a look

Comment (pfrazee) [reply src/view/com/composer/text-input/web/EmojiPicker.web.tsx:59]: Either way should work

Comment (ansh) [reply src/view/com/composer/text-input/web/EmojiPicker.web.tsx:59]: Doesn't seem like the React Native `ref` on a `View` has a `contains` method

Comment (pfrazee) [reply src/view/com/composer/text-input/web/EmojiPicker.web.tsx:59]: probably not worth sussing out the other approach, but I _believe_ there's a way to get the dom element out of the ref in the RNW build

**Review by pfrazee — APPROVED** [review]
👍 Seems to work!

**Pull request merged**